src/mocks/Bike.ts

```diff
@@ -0,0 +1,25 @@
+export const mockedImageUrls = [
+  'https://cremecycles.com/images/glowne/15.jpg',
+  'https://cremecycles.com/images/glowne/13.jpg',
+  'https://cremecycles.com/images/glowne/15.jpg'
+]
+
+const mockedBike: Bike = {
```

Comment:
Is this mock data being used somewhere?

Reply:
I think it's preferable to leave this mock data just for the candidate to have an idea of the data, without having to connect to the API